ouroboros-consensus/changelog.d/20250219_054827_nick.frisby_simplify_inject.md

```diff
@@ -1,24 +0,0 @@
-<!--
-A new scriv changelog fragment.
-
-Uncomment the section that is right (remove the HTML comment wrapper).
--->
-
-<!--
-### Patch
-
-- A bullet item for the Patch category.
-
--->
-<!--
-### Non-Breaking
-
-- A bullet item for the Non-Breaking category.
-
--->
-
-### Breaking
-
-- Make the type of `Nary.inject` more precise.
-  The old type involves oracular data, so any required changes downstream are almost certainly limited to testing code.
-  (I'm therefore tempted to list this as Patch instead --- violating the [PVP](https://pvp.haskell.org/) --- but the risk-reward seems prohibitive.)
```

Comment:
I have omitted this line from the collected changelog. Let me know if you actually want me to keep it

Reply:
I think it makes sense to remove it, good eye :+1: